clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/job/KubectlJobExecutorInterface.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2023 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.op.job;
+
+import com.google.common.collect.ImmutableList;
+import com.netflix.spinnaker.clouddriver.data.task.Task;
+import com.netflix.spinnaker.clouddriver.kubernetes.description.JsonPatch;
+import com.netflix.spinnaker.clouddriver.kubernetes.description.KubernetesPatchOptions;
+import com.netflix.spinnaker.clouddriver.kubernetes.description.KubernetesPodMetric;
+import com.netflix.spinnaker.clouddriver.kubernetes.description.manifest.KubernetesKind;
+import com.netflix.spinnaker.clouddriver.kubernetes.description.manifest.KubernetesManifest;
+import com.netflix.spinnaker.clouddriver.kubernetes.security.KubernetesCredentials;
+import com.netflix.spinnaker.clouddriver.kubernetes.security.KubernetesSelectorList;
+import io.kubernetes.client.openapi.models.V1DeleteOptions;
+import java.util.List;
+import javax.annotation.Nonnull;
+
+public interface KubectlJobExecutorInterface {
```

Comment:
let's rename it to `KubectlJobExecutor` and the implementation to `DefaultKubectlJobExecutor`